src/referencePackages/src/system.runtime/4.3.0/System.Runtime.csproj

```diff
@@ -0,0 +1,63 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.0;netstandard1.2;netstandard1.3;netstandard1.5;net462</TargetFrameworks>
+    <NuspecFile>$(ArtifactsBinDir)system.runtime/4.3.0/system.runtime.nuspec</NuspecFile>
+   </PropertyGroup>
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
```

Comment:
Might want to pull some of these properties out to a Directory.Build.props or dir.props if they'll be the same for future ref projects.

Reply:
Done.